src/ros_bridge/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM osrf/ros:foxy-desktop
+
+WORKDIR /opt/v-hsc-ros-bridge/
+
+# install ros build tools
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      python3-colcon-common-extensions libopencv-dev liblua5.2-dev protobuf-compiler libprotobuf-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+# clone ros package repo
+RUN git -C /opt clone \
+      -b main \
+      https://github.com/robocup-hl-tc/v-hsc-ros-bridge.git
+
+# install ros package dependencies
+RUN apt-get update && \
+    rosdep install -y \
+      --from-paths \
+        src/ros_bridge && \
```

Comment:
Maybe make this an absolute path? The `WORKDIR` depends on the parent image and at least in theory they could change that in an update of the image. But in any case it'd be a little bit more readable I suppose.

Reply:
Do you mean removing WORKDIR?

Reply:
No, I mean instead of using the relative path `src/ros_bridge` (which depends on the WORKDIR), you should use an absolute path.